projects/demo/src/modules/components/sheet-dialog/index.html

```diff
@@ -131,6 +143,15 @@
                 <code>--tui-offset</code>
                 )
             </ng-template>
+            <ng-template
+                documentationPropertyName="required"
+                documentationPropertyType="boolean"
+                [(documentationPropertyValue)]="required"
+            >
+                Cross click, overlay click or Escape click emits an error into
```

Comment:
Let's just say "Dismissing dialog ..." since there's no cross here and no escape on mobile